packages/odyssey-react/src/components/Select/Select.module.scss

```diff
@@ -12,12 +12,73 @@
 
 /* stylelint-disable color-no-hex, declaration-no-important, selector-class-pattern, max-nesting-depth, selector-max-compound-selectors, selector-no-qualifying-type, no-descending-specificity, selector-max-class, selector-max-type */
 
-/* - styles from _select.scss - */
-
 .root:not([data-type]) {
-  @include input-baseline;
+  @include border-radius(var(--BorderRadius));
 
   appearance: none;
+  display: block;
+  position: relative;
+  width: 100%;
+  max-width: var(--FontMaxWidth);
+  margin-block: var(--SpaceMarginBlock);
+  margin-inline: var(--SpaceMarginInline);
+  padding-block: var(--SpacePaddingBlock);
+  padding-inline: var(--SpacePaddingInline);
+  transition-property: border-color, background-color, box-shadow;
+  transition-duration: var(--TransitionDuration);
+  transition-timing-function: var(--TransitionTiming);
+  border-width: var(--BorderWidth);
+  border-style: var(--BorderStyle);
+  border-color: var(--ColorBorder);
+  background-color: var(--ColorBackground);
+  box-shadow: 0 0 0 0 cv("blue", "000");
+  color: var(--ColorText);
+  font-family: var(--FontFamily);
+  font-size: var(--FontSize);
+  line-height: var(--FontLineHeight);
+
+  &:hover,
+  &:focus,
+  &.is-ods-input-hover,
+  &.is-ods-input-focus {
```

Comment:
The hook let you commit these? It wouldn't let me yesterday.

Reply:
what do you mean by hook? stylelint static analysis?

This entire style module is a disaster because its trying to override via higher specificity in the cascade all the choices.js styles. [We have to disable all the meaningful static analysis checks](https://github.com/okta/odyssey/pull/1280) to do that which would explain what you're seeing.

Reply:
hmmm, now that you say that I should really go back through and remove some of these classes as they will no longer be relevant. They are never applied in the DOM any longer, even though the contents of `input-baseline` mixin still contained them! good feedback @francoislehoux-okta

Reply:
Either linting/hooks were preventing me from committing these "kebab-case" selectors for the TextArea PR

Reply:
@francoislehoux-okta updated in my latest patch to remove these thanks for review